docs/coreutils-experiments.md

```diff
@@ -60,7 +67,8 @@ This document is structured as a series of FAQs:
 
     Option `--randomize-fork` was used in the experiments but recently removed (commit [`5133b98`](https://github.com/klee/klee/commit/5133b98f1d989af94902366c6d02eb6447458aa1)).
 
-7.  What are the options closest to the ones above that work with the current version KLEE?
+
+8.  What are the options closest to the ones above that work with the current of version KLEE?
```

Comment:
shouldn't it be "current version of KLEE"?

Reply:
Why do you say that? :)  
Fixed now.